racket/src/ChezScheme/zlib/FAQ

```diff
@@ -20,7 +20,7 @@ The lastest zlib FAQ is at http://zlib.net/zlib_faq.html
  3. Where can I get a Visual Basic interface to zlib?
 
     See
-        * http://marknelson.us/1997/01/01/zlib-engine/
+        * https://marknelson.us/posts/1997/01/01/zlib-engine.html
```

Comment:
The changes under ChezScheme directory might be better submitted to https://github.com/cisco/ChezScheme/.